cookie_booths/models.py

```diff
@@ -332,29 +332,25 @@ def change_golden_status(self, is_golden_booth):
 
     def enable_freeforall(self):
         # If we're already enabled, nothing to do
-        if self.booth_day_freeforall_enabled and self.booth_day_enabled:
+        if self.booth_day_freeforall_enabled:
             return
 
         self.booth_day_freeforall_enabled = True
-        self.booth_day_enabled = True
 
         for block in BoothBlock.objects.filter(booth_day__id=self.id):
-            block.booth_block_enabled = True
             block.booth_block_freeforall_enabled = True
             block.save()
 
         self.save()
 
     def disable_freeforall(self):
         # If we're already disabled, nothing to do
-        if not self.booth_day_freeforall_enabled and not self.booth_day_enabled:
+        if not self.booth_day_freeforall_enabled:
             return
 
         self.booth_day_freeforall_enabled = False
-        self.booth_day_enabled = False
```

Comment:
So I was mainly thinking of like that 30min period before the booth opens where booths can't be reserved anymore for this code. Just curious - what's the assumption of what should actually happen to booths at this point?

Reply:
Booth enabled would be the only thing that matters. Especially since we're moving away from the FFA flag in later versions of the site. This will likely not even exist for the next cookie cycle.